esinstall/src/types.ts

```diff
@@ -12,6 +12,14 @@ export interface ImportMap {
   imports: {[packageName: string]: string};
 }
 
+export interface AbstractLogger {
+  debug: any;
+  log: any;
+  warn: any;
+  error: any;
```

Comment:
WDYT about typing this as `(message: string) => void`? That’d make things a little easier on the esinstall side, and if someone wanted to change this interface they could always just use `any` on their side

Reply:
👍 done!